src/components/community/Platforms/types.ts

```diff
@@ -0,0 +1,37 @@
+import type { Platforms as PlatformsType } from "temporaryData/types"
+
+type Error = { code: number; message: string } | null
+type State = "initial" | "loading" | "success" | Error
+type ModalProps = {
```

Comment:
It's not used anywhere, did it remain here accidentially?

Reply:
Yeah, used to have a wrapper component for the modal with just the modal body being separated, but I find this version much better. This type was for that component.